src/index.html

```diff
@@ -6,8 +6,47 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+      src="images/imac.jpeg"
+      alt="Apple iMac 27 Retina 5k Monoblock"
+      class="card__image"
+      >
```

Comment:
```suggestion
      <img
        src="images/imac.jpeg"
        alt="Apple iMac 27 Retina 5k Monoblock"
        class="card__image"
      >
```
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/65b7a4eb-e140-4ecb-a8be-bb71bda0da58)